components/brave_extension/extension/brave_extension/background/api/cosmeticFilterAPI.ts

```diff
@@ -1,120 +0,0 @@
-    if (chrome.runtime.lastError) {
-      console.warn('Unable to get cosmetic filter data for the current host', chrome.runtime.lastError)
-      return
-    }
-
-    if (frameId === 0) {
-      if (hide1pContent) {
-        resources.force_hide_selectors.push(...resources.hide_selectors)
-      } else {
-        informTabOfCosmeticRulesToConsider(tabId, resources.hide_selectors)
-      }
-
-      let styledStylesheet = ''
-      if (resources.force_hide_selectors.length > 0) {
-        styledStylesheet += resources.force_hide_selectors.join(',') + '{display:none!important;}\n'
-      }
-      for (const selector in resources.style_selectors) {
-        styledStylesheet += selector + '{' + resources.style_selectors[selector].join(';') + ';}\n'
-      }
-      chrome.tabs.insertCSS(tabId, {
-        code: styledStylesheet,
-        cssOrigin: 'user',
-        runAt: 'document_start'
-      })
-    }
-
-    shieldsPanelActions.cosmeticFilterRuleExceptions(tabId, frameId, resources.exceptions, resources.injected_script || '', resources.generichide)
-  })
-}
-
-// User generated cosmetic filtering below
-export const applyCSSCosmeticFilters = (tabId: number, hostname: string) => {
-  chrome.storage.local.get('cosmeticFilterList', (storeData = {}) => {
-    if (!storeData.cosmeticFilterList) {
-      if (process.env.NODE_ENV === 'shields_development') {
-        console.log('applySiteFilters: no cosmetic filter store yet')
-      }
-      return
-    }
-    if (storeData.cosmeticFilterList[hostname] !== undefined) {
-      storeData.cosmeticFilterList[hostname].map((rule: string) => {
-        if (process.env.NODE_ENV === 'shields_development') {
-          console.log('applying rule', rule)
-        }
-        chrome.tabs.insertCSS(tabId, { // https://github.com/brave/brave-browser/wiki/Cosmetic-Filtering
-          code: `${rule} {display: none !important;}`,
-          cssOrigin: 'user',
-          runAt: 'document_start'
-        })
-      })
-    }
-  })
-}
-
-export const removeAllFilters = () => {
-  chrome.storage.local.set({ 'cosmeticFilterList': {} })
-}
-
-export const addSiteCosmeticFilter = async (origin: string, cssfilter: string) => {
-  chrome.storage.local.get('cosmeticFilterList', (storeData = {}) => {
-    let storeList = Object.assign({}, storeData.cosmeticFilterList)
-    if (storeList[origin] === undefined || storeList[origin].length === 0) { // nothing in filter list for origin
-      storeList[origin] = [cssfilter]
-    } else { // add entry
-      storeList[origin].push(cssfilter)
-    }
-    chrome.storage.local.set({ 'cosmeticFilterList': storeList })
-  })
-}
-
-export const removeSiteFilter = (origin: string) => {
-  chrome.storage.local.get('cosmeticFilterList', (storeData = {}) => {
-    let storeList = Object.assign({}, storeData.cosmeticFilterList)
-    delete storeList[origin]
-    chrome.storage.local.set({ 'cosmeticFilterList': storeList })
-  })
-}
```

Comment:
Just a heads up, these functions and anywhere they're used are all for the right-click context menu -> `Brave` -> `Block element via selector` etc. on desktop.

We do want to merge them with the "Custom filters" in `brave://adblock` at some point, but that will require some migration work and until then we can't just remove them outright.

Reply:
ok, it looks like I will need to keep the extension in that case, I will make sure that we don't execute heuristic and don't pull anything from adblock list, but that part will work

Reply:
@antonok-edm I reverted that part, the menu functionality works as it used to be.